### ISSUE TITLE
Model outputs all background during evaluation (Dice = 0)

### ISSUE THREAD

**Issue by lawu-star** [hding2455/CaRTS#22]
Hi, thanks for your great work.

I encountered an issue during evaluation where the model seems to predict only background, and all metrics become zero:

mean: dice_tool: 0.000000
std: dice_tool: 0.000000
mean: nsd: 0.000000
std: nsd: 0.000000

I am using the official code and configuration, and the model weights are downloaded from the provided checkpoint (not trained by myself).

Some details I checked:

* The input images seem correctly normalized:
  FINAL IMAGE: around -2.11 to 2.64
* Ground truth masks are valid:
  FINAL GT: 0 / 1, unique values are tensor([0., 1.])
* The model weights are successfully loaded (not zero or uninitialized)

However, the predictions appear to be almost entirely background.

I am wondering if this could be related to:

* Whether the model outputs logits or sigmoid probabilities
* Any preprocessing mismatch between training and evaluation

Could you please clarify the expected output format of the model and the correct evaluation setting?

Thanks a lot for your help.

**Comment by hding2455** [comment]
Sorry for the late response. The model might be out-of-date, have you try to retrain wit the current code and test?

**Comment by hding2455** [comment]
The latest checkpoints are uploaded:

[Mask2Former](https://drive.google.com/file/d/1Jd6NORZuEcOEFesUDW4-3ddH7nux3w1T/view?usp=sharing)
[UNetPlusPlus](https://drive.google.com/file/d/1Y2zhKuOMiDkjlRjLHwwvzVxd5kwCEz2-/view?usp=drive_link)

You can also find them in the latest readme
